src/text-tags.js

```diff
@@ -60,6 +60,26 @@ function buildTagsFromTopic(config, tag, url = {}) {
   return topicMetaData;
 }
 
+function buildTagsFromAuthor(config, author, url = {}) {
+  if(isEmpty(author)) return;
+
+  const authorName = author.name;
+  const authorUrl = `${config['sketches-host']}/author/${author.id}`;
```

Comment:
If you look at how the tag seo is done, the publisher app's data loader passes in the path for the tag. Reason being, we don't yet enforce anywhere the structure of a tag or author page. The publisher is free to define any path for the tag/author page in their routes.js. Currently only the publisher app knows for a fact what the path is. Which is why, it should be the responsibility of the data loader to augment the author with the correct path info. That way the SEO package can simply prepend the domain name to the path and be done with it.

Reply:
Talk to @Athira001 or @asimqt . I believe one of them added the support for tag pages.

Reply:
Updated